examples/webgl_loader_multiple.html

```diff
@@ -0,0 +1,314 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - GLTFloader</title>
```

Comment:
Maybe title should be clearer. And if we support not only glb but also fbx, this title mismatches.

Reply:
Adjusted to three.js webgl - Multi-File Viewer (FBX and glTF)